src/lib/ride.ts

```diff
@@ -1,23 +1,33 @@
 import {db, RideModel} from '../database';
+import {Q} from '@nozbe/watermelondb';
+import EventEmitter from 'events';
 
-export function startRide() {
+const ev = new EventEmitter();
+
+export async function startRide() {
```

Comment:
should we maybe add explicit return types on these like `export function startRide(): Promise<RideModel> {}